src/config.rs

```diff
@@ -48,7 +49,7 @@ impl SearchConfig {
     ///
     /// * `room_id` - The unique id of the room.
     pub fn for_room(&mut self, room_id: &str) -> &mut Self {
-        self.room_id = Some(room_id.to_owned());
+        self.room_ids = Some(vec![room_id.to_owned()]);
```

Comment:
We need to allow multiple room ids to be set, no? That is we should allow the `for_room` method to be called multiple times, every time adding another room to the vec.

Reply:
Both seshat-node and radical-native just deserialize the json for the searchConfig, so I didn't add any methods.

But I've now added them for the test :smiley: